Userland/Libraries/LibWeb/Dump.cpp

```diff
@@ -599,7 +599,7 @@ void dump_font_face_rule(StringBuilder& builder, CSS::CSSFontFaceRule const& rul
     builder.append("sources:\n"sv);
     for (auto const& source : font_face.sources()) {
         indent(builder, indent_levels + 2);
-        builder.appendff("url={}, format={}\n", source.url, source.format.value_or("???"));
+        builder.appendff("url={}, format={}\n", source.url, source.format.has_value() ? source.format->bytes_as_string_view() : "???"sv);
```

Comment:
6 of one, half a dozen of the other, but since "???" is a short string, you could keep the `value_or` bit:
```c++
source.format.value_or(String::from_utf8_short_string("???"sv))
```